springboard_iogt/templates/base.jinja2

```diff
@@ -21,9 +21,11 @@
                 <span>
                     <a href="{{ 'locale_matched'|route_url(language=lang_code) }}">{{ lang_code|display_language_name }}</a>
                 </span>
-                <span> | </span>
+                {% if not loop.last %}<span> | </span>{% endif %}
             {% endfor %}
-            <span><a href="{{ 'locale_change'|route_url }}">&hellip;</a></span>
+            {% if display_languages|length + 1 < available_languages|length %}
```

Comment:
`display_languages` is `featured_languages` minus the currently selected language. So if your currently selected language is not in the `featured_languages` list, `display_languages` = `featured_languages`.